vector/src/main/java/im/vector/app/features/settings/devices/v2/list/OtherSessionsController.kt

```diff
@@ -71,7 +73,14 @@ class OtherSessionsController @Inject constructor(
                     sessionDescriptionDrawable(descriptionDrawable)
                     sessionDescriptionColor(descriptionColor)
                     stringProvider(this@OtherSessionsController.stringProvider)
+                    colorProvider(this@OtherSessionsController.colorProvider)
+                    drawableProvider(this@OtherSessionsController.drawableProvider)
```

Comment:
You can use `host` to have nicer code.
```suggestion
                    drawableProvider(host.drawableProvider)
```

applicable to other lines too.